src/wristpy/io/readers/readers.py

```diff
@@ -81,7 +81,7 @@ def geneActiv_loader(
 ) -> WatchData:
     """Load input data from GeneActiv .bin file using actfast.
 
-        This loads the acceleration, lux, battery voltage, and capsense data.
+        This loads the acceleration, lux, battery voltage, and temperature data.
```

Comment:
This snippet of code 

```python
time_tmp = pl.Series(subject1["timeseries"]["acceleration"]["datetime"])
time_actfast = pl.from_epoch(time_tmp, time_unit="ns").alias("time")
```

Occurs several times throughout this file for different datetimes. Consider turning it into a function e.g. `def actfast_time_to_pl_datetime(actfast_time: FILL_IN_TYPE) -> pl.Series`